### ISSUE TITLE
I propose to add a type for the Buffer file

### ISSUE THREAD

**Issue by deniszhigalov** [tus/tus-js-client#520]
I pass Buffer to Upload and it works for me, but I use typescript and it tells me that the type cannot be Buffer. If Upload takes Buffer type please fix it. Thank you for understanding.

**Comment by Acconut** [comment]
I agree, we should also cover the `Buffer` type and not only `Blob` from the browser: https://github.com/tus/tus-js-client/blob/1db985ac2a1531184a26e3e989de4a2d2f90a823/lib/index.d.ts#L9

Would you be willing to open a PR for this?

**Comment by major-winter** [comment]
The Buffer type is now included in `tus-js-client`.  I think this one can be closed.

**Comment by Acconut** [comment]
Thank you for reminding us that this has been resolved, @major-winter!
